packages/yourloops/components/dialogs/pdf-print-options.tsx

```diff
@@ -157,7 +156,7 @@ function DialogPDFOptions(props: DialogPDFOptionsProps): JSX.Element {
   }, [props.minDate, props.maxDate, customStartDate])
 
   const [openState, setOpenState] = React.useState(false)
-  const [pdfOptions, setPDFOptions] = React.useState<PrintPDFOptions>(getDatesFromPreset(defaultPreset || DEFAULT_PRESET, minDate, maxDate))
+  const [pdfOptions, setPDFOptions] = React.useState<PrintPDFOptions>(getDatesFromPreset(defaultPreset || DEFAULT_PRESET, minDate, maxDate, 'pdf'))
```

Comment:
It would be best to manipulate a shared `const` here (defined in an enum, or at the top of the file) instead of a raw string that can be changed easily

Reply:
This is actually a type. :slightly_smiling_face: 
Defined like this:
`export type OutputFormat = 'pdf' | 'csv'`
So it can only have the value 'pdf' or 'csv', otherwise the compilation fails.
I personally prefer enums but I didn't want to change everything...